BazarRestAPI/Controllers/UsersController.cs

```diff
@@ -41,13 +54,21 @@ public ActionResult<User> Get(int id)
         {
             try
             {
-                return Ok(_userService.GetByID(id));
+                User user = _userService.GetByID(id);
```

Comment:
You use this multiple times. Maybe you could make a private method to do this so you can reuse it.